docs/identity-platform/jwt-claims-customization.md

```diff
@@ -275,83 +275,43 @@ To run this script, you need:
 > The private key must be in PKCS#12 format since Microsoft Entra ID doesn't support other format types. Using the wrong format can result in the error "Invalid certificate: Key value is invalid certificate" when using Microsoft Graph to PATCH the service principal with a `keyCredentials` containing the certificate information.
 
 ```powershell
-$fqdn="fourthcoffeetest.onmicrosoft.com" # this is used for the 'issued to' and 'issued by' field of the certificate
-$pwd="mypassword" # password for exporting the certificate private key
-$location="C:\\temp" # path to folder where both the pfx and cer file will be written to
+##########################################################
+# Replace the variables below with the appropriate values 
+
+$fqdn="yourDomainHere" # This is used for the 'issued to' and 'issued by' field of the certificate
+$pwd="password" # password for exporting the certificate private key
+$tenantId   = "franciscogom.es" # Replace with Tenant ID
+$appObjId = "384bd956-daaa-4ac3-b558-a1e5ceb15bd2" # Replace with the Object ID of the App Registration
```

Comment:
Is it the Application (client) id or the Object ID here? They are two different things. Is the tenant ID a GUID, or a domain?

Replacing GUIDs/values as they are sensitive info

```suggestion
$tenantId   = "aaaabbbb-0000-cccc-1111-dddd2222eeee" # Replace with your Tenant ID
$appObjId = "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" # Replace with the Object ID of the App Registration
```